mne/io/brainvision/brainvision.py

```diff
@@ -63,6 +63,9 @@ class RawBrainVision(BaseRaw):
 
     Notes
     -----
+    If the BrainVision header file contains impedance measurements, these may be
+    accessed using ``raw.impedances`` after reading using this function.
```

Comment:
Question: I see that this attribute was discussed and added many years ago in #7974, but I thought that attributes that don't survive I/O roundtrips aren't supposed to be added?

e.g. https://github.com/mne-tools/mne-python/issues/11825#issue-1816752541

Reply:
Note that I actually stumble upon this while adding the ANT reader, and thus I added a similar `ìmpedance` attribute in the ANT reader. Maybe it should be removed as well and live in `antio`.